### ISSUE TITLE
[feature]: admonish discriminator

### ISSUE THREAD

**Issue by ShaunSHamilton** [tommilligan/mdbook-admonish#87]
I have encountered two issues with the current methods of indicating `adomonish` blocks:

1) Prettier breaks "super-blocks"

When wrapping codeblocks in an `admonish` block, prettier changes the current discriminator (`~~~`) to the more common quadruple backtick (\`\`\`\`):

**Before Prettier**

````markdown
~~~admonish
```json
{}
```
~~~
````

**After Prettier**

`````markdown
````admonish
```json
{}
```
````
`````

2) I miss the syntax highlighting within `admonish` blocks

<img width="170" alt="image" src="https://user-images.githubusercontent.com/51722130/233710685-9abc9805-298c-461f-89f1-f13e2e190648.png">

---

**Suggestions**

a) Use an alternate, completely unique, discriminator:

<details><summary>Alternatives</summary>

````markdown
:::admonish
```json
{}
```
:::

!!!admonish
```json
{}
```
!!!
````

</details> 

b) Accept quadruple backticks:
https://github.com/tommilligan/mdbook-admonish/blob/e8813eb104734881f7aa313d09178510da42c9ee/src/lib.rs#L223

_bonus points if it is made possible to wrap an admonish block within an admonish block_

_sorry for all the noise from my end_

**Comment by tommilligan** [comment]
Thanks for the bug report! I don't want to use an alternative discriminator as I'm relying on a markdown parser to do the heavy lifting. I've merged #88 which should handle a bunch more cases from the spec; if there are any I'm missing they can be opened as future bugs.

Hopefully this fixes your original case (now added as a test case)

**Comment by ShaunSHamilton** [comment]
Thank you for the quick fix.

I am sad about the missing syntax highlighting. However, I understand and agree with not straying from spec just for that.
